keyboards/keychron/q2/halconf.h

```diff
@@ -16,11 +16,11 @@
 
 /*
  * This file was auto-generated by:
- *    `qmk chibios-confmigrate -i keyboards/nk65/halconf.h -r platforms/chibios/QMK_PROTON_C/configs/halconf.h`
+ *    `qmk chibios-confmigrate -i keyboards/keychron/q2/halconf.h -r platforms/chibios/boards/QMK_PROTON_C/configs/halconf.h`
```

Comment:
Just delete this whole comment block. It's injected when it's been migrated, but serves no purpose when it's a new board. No need to keep it.